packer/tests/Gemfile

```diff
@@ -0,0 +1,4 @@
+source "https://rubygems.org"
```

Comment:
Wait, no, I take back my thumbs up. My understanding is that checking in the Gemfile.lock is best practice when you are building an application, have a lot of dependencies and are interested in getting precise versions from specific sources. 

In our case, we're using a gemfile here as a convenience, so we can do `bundle install` instead of having to do `gem install rake` and then `gem install serverspec`. Furthermore, `bundle install` is happening as a provisioning step, so the Gemfile.lock exists only on the created machine images. If we wanted to include it in source control, we'd have to copy them from our images onto our local working directory or run bundler again locally. For these reasons, I am advocating that we do not include Gemfile.lock.